cluster/gce/windows/k8s-node-setup.psm1

```diff
@@ -1548,18 +1580,18 @@ function Install-LoggingAgent {
       ("$STACKDRIVER_ROOT\LoggingAgent\Main\pos\winevtlog.pos\worker0\" +
        "storage.json")
 
-  if (Test-Path $STACKDRIVER_ROOT) {
+  if (IsLoggingAgentInstalled) {
     # Note: we should reinstall the Stackdriver agent if $REDO_STEPS is true
     # here, but we don't know how to run the installer without it prompting
     # when Stackdriver is already installed. We dumped the strings in the
     # installer binary and searched for flags to do this but found nothing. Oh
     # well.
-    Log-Output ("Skip: $STACKDRIVER_ROOT is already present, assuming that " +
-                "Stackdriver logging agent is already installed")
-    Restart-LoggingAgent
+    Log-Output ("Skip: Stackdriver logging agent is already installed")
     return
   }
 
+  Cleanup-LoggingAgent
```

Comment:
Please add a comment explaining why we need to call this here - we need to explain that after a crash, the Stackdriver service could be missing but its files will still be present. Right now it looks strange that the check above determined that the agent is not installed, but we're cleaning it up for some reason.

Reply:
Done

Reply:
You can remove the last two lines :)

```
  # Right now it looks strange that the check above determined that the agent is not installed,
  # but we're cleaning it up for some reason.
```

Reply:
Done :)